async/format.browser.js

```diff
@@ -1,5 +1,8 @@
 module.exports = function (random, alphabet, size) {
+  // TODO
   var mask = (2 << 31 - Math.clz32((alphabet.length - 1) | 1)) - 1
+  // -~i => i + 1 if n is integer number
```

Comment:
Do we need a note about `integer` here?

Reply:
Just found the comment about it in https://github.com/ai/nanoid/pull/161#issue-359831645
It can be important

Reply:
@ai Do you mind if I remain it as is?

Reply:
Yeap, just move it below. More relevant case (float) first.

Reply:
Done ✅